virkarekisteri/virkarekisteri
# VIR-37 Eligibilty

Added eligibility fields to creating new post

How to test:
1. Rebuild database
2. Open virkarekisteri
3. Open Uusi virka modal
4. See that the fields are there
5. Fill sometihng to these fields and submit, should be added to the position
